lib/thredded_create_app/tasks/setup_app_skeleton.rb

```diff
@@ -61,11 +67,26 @@ def add_favicon_and_touch_icons
                          ERB
       end
 
-      def configure_assets
-        copy 'setup_app_skeleton/manifest.js',
-             'app/assets/config/manifest.js'
+      def add_javascripts_webpack
+        append_to_file 'app/javascript/packs/application.js', <<~JS
+          require('monkey-patch-turbolinks.js');
+          require('app');
+        JS
+        copy 'setup_app_skeleton/javascript_webpack/monkey-patch-turbolinks.js',
+             'app/javascript/monkey-patch-turbolinks.js'
+        %w[app index theme time_ago].each do |file|
+          copy "setup_app_skeleton/javascript_webpack/app/#{file}.js",
+               "app/javascript/app/#{file}.js"
+        end
+        git_commit 'Add app JavaScript'
+      end
 
+      def configure_sprockets
+        replace 'app/assets/config/manifest.js',
+                "//= link_directory ../stylesheets .css\n",
+                ''
         append_to_file 'config/initializers/assets.rb', <<~RUBY
+          Rails.application.config.assets.precompile += %w[manifest.js]
```

Comment:
sprockets 3.x only?

Reply:
Removed instead, the generator doesn't need to support older Rails